src/openssl/mod.rs

```diff
@@ -109,8 +116,18 @@ pub struct ECVRF {
     qlen: usize,
     // 2n = length of a field element in bits rounded up to the nearest even integer
     n: usize,
-    // Cofactor of the curve
-    cofactor: u8,
+}
+
+impl Debug for ECVRF {
```

Comment:
I support this implementation for the sake of not adding more dependencies to this library.

For what it's worth, there are a couple of crates providing macros that achieve the same goal of excluding some fields from the `struct` so the `Debug` implementation can be automatically `Derive`d:
- [debug_stub_derive](https://crates.io/crates/debug_stub_derive)
- [derivative](https://crates.io/crates/derivative)